kfdefs/overlays/moc/smaug/opf-trino-stage/vpa/vpa-custom-resource.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  namespace: opf-trino-stage
+  name: vpa-trino-stage
+  labels:
+    app: vpa-trino-stage-v1
+spec:
+  resourcePolicy:
+    containerPolicies:
+      - containerName: trino-worker
+        maxAllowed:
+          cpu: 3000m
+          memory: 3Gi
+        minAllowed:
+          cpu: 400m
```

Comment:
Have you tested that worker's will run queries at `400m` ?

Reply:
Yes, when we tested with long queries for around 10 min, the cpu utilization was around 200m. As we might run longer queries for longer duration, we set the min allowed as 400m to be on the safer side. 

As we don't have cpu usage most of the time, VPA automatically drops the cpu requests to 25m. And when there is a heavy load, it crashes because of low cpu. So, if we set the minAllowed, it tells VPA not to go below that value ever.